### PR TITLE
Improve Footer Component

### LINKED ISSUE #5: Footer Component
The footer component should contain information and links such as imprint, copyright, links to Redi School, etc.

### PR DESCRIPTION
**Pull request by mojvaf**
- Improves layout of footer component
- Adds (stub) links to footer navigation
- Adds links to socials
- Fixes #5

### CODE REVIEW

**Comment by lopis** [comment]
I'm confused, why did you create a new pull request?

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -1,12 +1,62 @@
...
+      color: theme.palette.secondary.main,
+      fontFamily: 'Arial',
+      fontSize: '1rem',
+      textDecoration:'none'
+
+    },
+    icon:{
+      color:'theme.palette.secondary.main',
+      '&:hover':{
+        fill:'theme.palette.primary.dark',
```
Comment (lopis) [inline src/components/Footer.tsx]: `'theme.palette.primary.dark'` is a string. This won't produce the value you want.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -1,12 +1,62 @@
...
-      <Container>
-        Built with ❤️ by ReDI volunteers, teachers, and students
-      </Container>
+      <Grid container justify='space-around'  className={classes.mainContainer}>
+        <Grid item className={classes.link} >
+          <Hidden mdDown>
+            <Grid container justify='space-around' spacing={2}>
+              <Grid item  /*component={Link} to="/"*/ >Home</Grid>
+              <Grid item  /*component={Link} to="/About"*/ >Redi School</Grid>
+              <Grid item  /*component={Link} to="/Redi"*/ >Copyright</Grid>
```
Comment (lopis) [inline src/components/Footer.tsx]: Break long lines.

```suggestion
              <Grid item  /*component={Link} to="/"*/ >
                Home
              </Grid>
              <Grid item  /*component={Link} to="/About"*/ >
                Redi School
              </Grid>
              <Grid item  /*component={Link} to="/Redi"*/ >
                Copyright
              </Grid>
```

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -1,12 +1,62 @@
...
+            </Grid>
+          </Hidden>
+        </Grid>
+        <Grid item className={classes.link} >
+          <Grid container>
+            <Grid item /*component={"a"} herf="http://www.facebook.com" rel="noopener noreferrer" target="_blank"*/>
+              <Facebook className={classes.icon}/>
+            </Grid>
+            <Grid item /*component={"a"} herf="http://www.facebook.com" rel="noopener noreferrer" target="_blank"*/>
+              <Tweeter className={classes.icon}/>
```
Comment (lopis) [inline src/components/Footer.tsx:60]: Did you mean "Twitter"?

File: src/components/Footer.tsx [hunk 4]
```diff
@@ -1,12 +1,62 @@
...
+            <Grid container justify='space-around' spacing={2}>
+              <Grid item  /*component={Link} to="/"*/ >Home</Grid>
+              <Grid item  /*component={Link} to="/About"*/ >Redi School</Grid>
+              <Grid item  /*component={Link} to="/Redi"*/ >Copyright</Grid>
+            </Grid>
+          </Hidden>
+        </Grid>
+        <Grid item className={classes.link} >
+          <Grid container>
+            <Grid item /*component={"a"} herf="http://www.facebook.com" rel="noopener noreferrer" target="_blank"*/>
```
Comment (lopis) [inline src/components/Footer.tsx]: Why are the links to social media commented? I think we can already add links to the icons.

**Review by lopis — CHANGES_REQUESTED** [review]
Hello! Please see my comments below.
Also, your `yarn.lock` file is still causing conflicts with the `main` branch; you can't merge if you don't solve them (just run `yarn` to install all dependencies and update the lock file). Finally, if you're done with #27 please close it.

**Comment by lopis** [comment]
Also, I have edited the title and the description of your pull request. It's important for us reviewers that your title and your description clearly describe the code you are submitting.

**Comment by mojvaf** [comment]
> Hello! Please see my comments below.
> Also, your `yarn.lock` file is still causing conflicts with the `main` branch; you can't merge if you don't solve them (just run `yarn` to install all dependencies and update the lock file). Finally, if you're done with #27 please close it.

Hello!  Today I was very busy but I will work on it tomorrow morning. Also, there is a bug that I am unbale to fix it. The problem is when hover on icons the color won't change. Could someone check it? Thank you

File: src/components/Footer.tsx (hunk 1 above)
Comment (mojvaf) [reply src/components/Footer.tsx]: what should I do then? I changed 'theme.palette.primary.dark' to something like 'green' and everything works with no bug but I need to use theme color. Any suggestions?

File: src/components/Footer.tsx (hunk 3 above)
Comment (mojvaf) [reply src/components/Footer.tsx:60]: Yes I meant Twitter. I corrected it. Thanks

File: src/components/Footer.tsx (hunk 4 above)
Comment (mojvaf) [reply src/components/Footer.tsx]: I corrected it too

File: src/components/Footer.tsx (hunk 1 above)
Comment (lopis) [reply src/components/Footer.tsx]: It's not supposed to be a string, but a value, like so: `fill: theme.palette.primary.dark,`

**Comment by lopis** [comment]
If you fix the issues I described in the comments related to the theme, that issue with the icons will also be fixed.

File: src/components/Footer.tsx (hunk 1 above)
Comment (mojvaf) [reply src/components/Footer.tsx]: Thank you

File: src/components/Footer.tsx (hunk 3 above)
Comment (lopis) [reply src/components/Footer.tsx:60]: I still see Tweeter

**Comment by lopis** [comment]
Thanks for the patiente @mojvaf! I think it looks fine now. I pushed some extra changes because I made a mistake in the code-style auto-fixer, but now it looks ok.

**Review by lopis — APPROVED** [review]

**Comment by lopis** [comment]
@all-contributors let's add @mojvaf for code!

**Comment by lopis** [comment]
@all-contributors let's add @mojvaf for code!

**Comment by mojvaf** [comment]
> @all-contributors let's add @mojvaf for code!

Thanks

**Pull request merged**